### PR TITLE
view: stop crashing when license field is an object

### PR DESCRIPTION
**Pull request by zkat**

### CODE REVIEW

File: lib/view.js [hunk 1]
```diff
@@ -208,7 +209,9 @@ function prettyView (packument, manifest) {
       description: manifest.description,
       deprecated: manifest.deprecated,
       keywords: (packument.keywords || []).map(color.yellow),
-      license: manifest.license || manifest.licence || 'Proprietary',
+      license: typeof licenseField === 'string'
+        ? licenseField
+        : (licenseField.type || 'Proprierary'),
```
Comment (joeledwards) [inline lib/view.js]: Here and 🍎 `Proprierary` should be `Proprietary`

File: lib/view.js [hunk 2]
```diff
@@ -200,6 +200,7 @@ function prettyView (packument, manifest) {
     })
     const unpackedSize = manifest.dist.unpackedSize &&
       byteSize(manifest.dist.unpackedSize)
+    const licenseField = manifest.license || manifest.licence || 'Proprierary'
```
Comment (joeledwards) [inline lib/view.js]: 🍎

**Review by joeledwards — CHANGES_REQUESTED** [review]

**Comment by zkat** [comment]
@joeledwards wanna re-review?

**Review by joeledwards — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**